pnc-core/src/test/java/org/jboss/pnc/core/test/mock/EnvironmentDriverMock.java

```diff
@@ -1,19 +1,31 @@
 package org.jboss.pnc.core.test.mock;
 
-import org.jboss.pnc.model.Environment;
-import org.jboss.pnc.spi.environment.EnvironmentDriver;
-
-import javax.enterprise.context.ApplicationScoped;
 import java.lang.invoke.MethodHandles;
 import java.util.logging.Logger;
 
+import javax.enterprise.context.ApplicationScoped;
+
+import org.jboss.pnc.model.Environment;
+import org.jboss.pnc.spi.environment.EnvironmentDriver;
+import org.jboss.pnc.spi.environment.RunningEnvironment;
+import org.jboss.pnc.spi.environment.exception.EnvironmentDriverException;
+
 @ApplicationScoped
 public class EnvironmentDriverMock implements EnvironmentDriver {
 
     private static final Logger logger = Logger.getLogger(MethodHandles.lookup().lookupClass().getName());
 
     @Override
-    public void buildEnvironment(Environment buildEnvironment) {
+    public RunningEnvironment buildEnvironment(Environment environment, String dependencyUrl, String deployUrl)
+            throws EnvironmentDriverException {
+        // TODO Auto-generated method stub
```

Comment:
That are mocks, which was there a longer time ago. I didn't want to make this patch too big. I'll remove it after integrating this driver with JenkinsBuildDriver